modules/light-clients/08-wasm/types/vm.go

```diff
@@ -287,25 +295,25 @@ func getEnv(ctx sdk.Context, contractAddr string) wasmvmtypes.Env {
 	return env
 }
 
-func humanAddress(canon []byte) (string, uint64, error) {
+func humanizeAddress(canon []byte) (string, uint64, error) {
 	return "", 0, errors.New("humanAddress not implemented")
 }
 
-func canonicalAddress(human string) ([]byte, uint64, error) {
+func canonicalizeAddress(human string) ([]byte, uint64, error) {
 	return nil, 0, errors.New("canonicalAddress not implemented")
 }
 
 // checkResponse returns an error if the response from a sudo, instantiate or migrate call
 // to the Wasm VM contains messages, events or attributes.
-func checkResponse(response *wasmvmtypes.Response) error {
+func checkResponse(response *wasmvmtypes.ContractResult) error {
```

Comment:
```suggestion
func checkResponse(response *wasmvmtypes.Response) error {
```

I'd keep this type can adjust the callers